pkg/dashboards/dashboards_test.go

```diff
@@ -288,34 +287,24 @@ var (
 
 func TestListDashboards(t *testing.T) {
 	t.Parallel()
-	apm := NewTestDashboards(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	client := NewTestDashboards(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This should use the `newMockResponse`

Reply:
Yup, I'll get this consolidated in a fast-follow PR